en/orm/saving-data.rst

```diff
@@ -991,3 +993,18 @@ A bulk-update will be considered successful if 1 or more rows are updated.
 
     updateAll will *not* trigger beforeSave/afterSave events. If you need those
     first load a collection of records and update them.
+
+
+``updateAll()`` is for convenience only. You can use this more flexible interface as well::
+
+    // Publish all the unpublished articles.
+    function publishAllUnpublished()
+    {
+        $this->query()
+            ->update()
+            ->set(['published' => 'true])
+            ->where(['published' => 'false'])
+            ->execute();
+    }
+    
+Also see: :doc:`Updating Data</orm/query-builder#updating-data>`.
```

Comment:
@ionas This won't work. If you wan't to link to a specific part, you need to use something like ``:ref:query-builder-updating-data`` and on top of the paragraph header ``.. _query-builder-updating-data:`` (it will automatically use the header content)